lib/macho/load_commands.rb

```diff
@@ -528,6 +542,35 @@ class DylibCommand < LoadCommand
       # @return [Integer] the library's compatibility version number
       field :compatibility_version, :uint32
 
+      def self.new_from_bin(view)
```

Comment:
Minor: it'd be nice to have a comment here explaining why we override `new_from_bin` -- just a summary of what you put in the PR desc would be great 🙂